### DISCUSSION TITLE
Server sandbox setting for 'frozen' submarine.

### DISCUSSION THREAD

**Discussion by Ek-Videogames** [FakeFishGames/Barotrauma#15161] [Ideas and suggestions]
Like how "Outpost PVP" works, it would be nice for server operators to have a setting in sandbox to enable freezing the submarine structure by default, such as for "outpost" themed submarines and custom PVP maps.

Also this could be better than the standard freeze command for more reasons than just convenience, as now that "shuttles" work in outpost PVP that allows shuttles & moving parts like 'elevators' to function while the main body is stationary.

**Comment by myrukun** [comment]
me and many others would benefit from this, as the lock command locks all movement, even shuttles. So having a lock command or setting that only locks the main sub, while maintaining the ability to use shuttles would be nice.
